index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ferizad's portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="homepage">
+        <header>
+            <h1 class="logo">Welcome</h1>
+            <img class="hamburger" src="media\Union.svg" alt="hamburger" width="20px" height="20px">
```

Comment:
- Hey, also, you should not use inline styling in the HTML file as this is not considered a good practice. kindly remove the width and height you've added and style it using CSS.
- The hamburger should be wrapped in a `nav` tag as the `nav` tag contains the navigation content in the document to improve the web accessibility of your website. kindly add the nav tag.